lintplus_php.lua

```diff
@@ -0,0 +1,48 @@
+-- lite-xl 1.16
+
+-- PHP lint plugin for lint+
+
+--- CONFIG ---
+
+-- config.lint.php_args: {string}
+--   passes the specified arguments to php
+
+--- IMPLEMENTATION ---
+
+local lintplus = require "plugins.lintplus"
+
+local redirection = ""
+if PLATFORM == "Linux" then
+  redirection = "2> /dev/null"
+end
```

Comment:
I just noticed this while reviewing the code a second time. I'm a afraid that redirection thingy isn't going to work properly when lintplus is in async mode, as then the arguments are not concatenated into a shell command, but rather run as a process directly via a native library.

Is there any extra output PHP has that goes into stderr? If so, does it match the pattern string you specified in the interpreter?

Reply:
Removed the redirection, the purpose was to reduce the amount of output that lite shows on the terminal.